.github/workflows/pytest.yml

```diff
@@ -2,11 +2,21 @@ name: run_tests
 
 on:
   workflow_call:
+    inputs:
+      workflow-call-ckeck:
+        description: 'set to true if calling from a workflow. So we can check the trigger and set the secrets accordingly'
+        required: true
+        type: boolean
+    secrets:
+      MAuserPAT-workflow-call:
```

Comment:
Is there a specific logic behind the naming

Reply:
Machine User Private Access Token . this part is the same as in the code it is used. the workflow-call is because i have to differentiate between "normal secrets" and "secrets given through a workflow call"

Reply:
ok, it was also more about the MAuserPAT Part. Maybe add a comment with the full name where the secret name is defined first.